src/content/configuration/output.mdx

```diff
@@ -1145,6 +1146,52 @@ If we specify `output.library.name` with `type: commmonjs2`, the return value of
 
 T> Wondering the difference between CommonJS and CommonJS2 is? While they are similar, there are some subtle differences between them that are not usually relevant in the context of webpack. (For further details, please [read this issue](https://github.com/webpack/webpack/issues/1114).)
 
+##### type: 'commonjs-static'
```

Comment:
We also mention versions in which this feature will be available via a badge.

```suggestion
##### type: 'commonjs-static' <Badge text='5.66.0+' />
```

Reply:
Avoid it in heading. We can insert it right after the heading.

Reply:
done